Tests/Marketplace/search_and_install_packs.py

```diff
@@ -332,11 +332,14 @@ def install_packs(client: demisto_client,
             result_object = ast.literal_eval(response_data)
             message = result_object.get('message', '')
             raise Exception(f'Failed to install packs - with status code {status_code}\n{message}')
-    except Exception:
-        logging.exception('The request to install packs has failed.')
+    except Exception as e:
+        logging.exception(f'The request to install packs has failed. Additional info: {str(e)}')
```

Comment:
```suggestion
        logging.exception(f'The request to install packs has failed. Additional info: {e.args[0]}')
```
Do we want to change it?

Reply:
I think we will leave it that way as I saw some odd behavior when using `e.args[0]`.